File: problems/fizzbuzz/fizzbuzz.rb
def transFizzBuzz(i)
   if ((i % 3).zero? && (i % 5).zero?) 
      return "FizzBuzz"
   end
   if (i % 3).zero? 
      return "Fizz"
   end
   if (i % 5).zero? 
      return "Buzz"
   end
   return i
end

(1..100).step(1) do |i|
   puts(transFizzBuzz(i))
end

File: problems/fizzbuzz/fizzbuzz.go
package main

import (
   "fmt"
   "strconv"
)

func transFizzBuzz(i int) (string) {
   if (i % 3) == 0 && (i % 5) == 0 {
      return "FizzBuzz"
   }
   if (i % 3) == 0 {
      return "Fizz"
   }
   if (i % 5) == 0 {
      return "Buzz"
   }
   return strconv.Itoa(i)
}

func main() {
   for i := 1; i < 101; i++ {
      fmt.Println(transFizzBuzz(i))
   }
}
